integreat_cms/cms/fixtures/test_data.json

```diff
@@ -355,6 +355,23 @@
       "message_partial_live_content_not_available": "Ein Teil von der Seite ist nicht in Ihrer Sprache. Der Teil ist aber in diesen Sprachen verfügbar:"
     }
   },
+  {
+    "model": "cms.language",
+    "pk": 7,
+    "fields": {
+      "slug": "hidden",
```

Comment:
I think it's ok to add a new language to allow a few more edge cases in the test data which should increase test coverage in the long term.
Probably that's a bit nit-picky, but I would prefer a "real" language in the test data, e.g. Spanish, French or something (and set a flag).

Reply:
We discussed that and decided that it would make sense to separate test cases, especially here to make a dummy language that would not be used for real in the future and thus be less prone to accidental activation of the `visible` setting, breaking this test from a seemingly unrelated context.